### DISCUSSION TITLE
Welcome to tamil_ocr Discussions!

### DISCUSSION THREAD

**Discussion by gnana70** [gnana70/tamil_ocr#12] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kiranbodipati** [comment]
Hey, could you give steps to finetune/train this model for custom data? Thanks

**Reply by gnana70** [reply]
Sure, will write an article on the same and share it

**Reply by kiranbodipati** [reply]
Thank you! I'm primarily looking to finetune it for tamil document images

**Reply by kiranbodipati** [reply]
Can you also share the Python Version used? And also share the setup you have used? I am facing dependency issues. 

Thanks

**Reply by gnana70** [reply]
Python 3.10

**Reply by gnana70** [reply]
Check out the colab for environment setup. https://colab.research.google.com/drive/11QPPj3EmpoIqnpuIznKeP1icxvVOjfux?usp=sharing

**Reply by gnana70** [reply]
@kiranbodipati , you don't need to train the text recognition model for document text, but need to train the craft text detection model. You can use this article ( https://pub.towardsai.net/how-to-fine-tune-the-craft-model-in-easyocr-f9fa0ac5cc9d)  to fine-tune craft text detection model

**Comment by ulagappan** [comment]
Thanks, this is great efforts. It will be useful to convert old thamzh books in pdf to text

**Comment by adhavan18** [comment]
Hey? Can you transcript old tamil manuscripts? I've also been getting a torch._C module error even when i imported all the necessary ones. Can you please guide and assist? Thanks

**Reply by gnana70** [reply]
Sure, can you share the image that you have used

**Comment by adhavan18** [comment]
![olai](https://github.com/gnana70/tamil_ocr/assets/99407456/343e9707-6447-4909-af2d-b390c991223d)
![olai2](https://github.com/gnana70/tamil_ocr/assets/99407456/02e3f513-d635-43b2-86e1-f9473f782947)
![olai4](https://github.com/gnana70/tamil_ocr/assets/99407456/d17e4173-b1be-4ad2-8687-276813a34c0e)
![olai5](https://github.com/gnana70/tamil_ocr/assets/99407456/9ae1aa32-cb84-47dc-98c8-bd4232d53941)

is it possible to transcript olai suvadi like all this into a readable txt file?

**Reply by gnana70** [reply]
currently performance won't be upto the mark for olai suvadi.
